[2024-07-24T10:51:35.653+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-24T10:51:35.674+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag_v2.second_task manual__2024-07-24T10:51:34.263647+00:00 [queued]>
[2024-07-24T10:51:35.679+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag_v2.second_task manual__2024-07-24T10:51:34.263647+00:00 [queued]>
[2024-07-24T10:51:35.679+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-24T10:51:35.691+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): second_task> on 2024-07-24 10:51:34.263647+00:00
[2024-07-24T10:51:35.694+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=244) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-24T10:51:35.695+0000] {standard_task_runner.py:64} INFO - Started process 245 to run task
[2024-07-24T10:51:35.695+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'first_dag_v2', 'second_task', 'manual__2024-07-24T10:51:34.263647+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpl5g7g5nx']
[2024-07-24T10:51:35.696+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask second_task
[2024-07-24T10:51:35.718+0000] {task_command.py:426} INFO - Running <TaskInstance: first_dag_v2.second_task manual__2024-07-24T10:51:34.263647+00:00 [running]> on host 312fcc3259a2
[2024-07-24T10:51:35.760+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Pattakit' AIRFLOW_CTX_DAG_ID='first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-24T10:51:34.263647+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-24T10:51:34.263647+00:00'
[2024-07-24T10:51:35.761+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-24T10:51:35.766+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-24T10:51:35.767+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am the second task and will run after first_tasj']
[2024-07-24T10:51:35.770+0000] {subprocess.py:86} INFO - Output:
[2024-07-24T10:51:35.771+0000] {subprocess.py:93} INFO - hey, I am the second task and will run after first_tasj
[2024-07-24T10:51:35.771+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-24T10:51:35.772+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-24T10:51:35.787+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=first_dag_v2, task_id=second_task, run_id=manual__2024-07-24T10:51:34.263647+00:00, execution_date=20240724T105134, start_date=20240724T105135, end_date=20240724T105135
[2024-07-24T10:51:35.797+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-07-24T10:51:35.804+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-24T10:51:35.805+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
